codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/EventSourcedEntitySourceGeneratorSuite.scala

```diff
@@ -111,7 +111,7 @@ class EventSourcedEntitySourceGeneratorSuite extends munit.FunSuite {
           |class MyEntityHandler(entity: MyEntity) extends EventSourcedEntityHandler[MyState, MyEntity](entity) {
           |  def handleEvent(state: MyState, event: Any): MyState = {
           |    event match {
-          |      case SetEvent =>
+          |      case _: SetEvent =>
```

Comment:
oh, here we can do

```scala
case evt: SetEvent => entity.setEvent(state, evt)
```

and remove the instance of :-)

Reply:
We have probably coded like that for the other components as well and we can clean up. Maybe later. Don't need to be now.

Reply:
argh I've become too mechanical LOL